Questbook updates including lategame progression changes

StaffiX opened the pull request:
- Updated EEC to accommodate the weapon controller slot change
- Make zpm+ circuit assembler optional due to CAL CA mode
- Add Baryonic Matter, Dimensionally Shifted Superfluid and Protohalkonite Quest
- Moved Metastable Oganesson and Deep Dark to UIV
- Moved Plasma Mixer next to Proto-Halkonite
- Fixed some (but not all) capitalization
- Removed UMV field gen from general component quest
- Updated Indium bee quest description
- Updated LNR and Naq Refinery description
- Updated plasma line description
- Removed fast mode mention from XL Steam Turbine
- Updated LP questline dependencies
- Removed mention of steam usage in ruthenium quest
- Added raw ore type to tengam quest
- Removed MCR 1 tick limit mention
- Replaced LPF mention with Industrial Autoclave in Crystal Chip quest
- Replaced UV energy hatch with UHV in EIC since every recipe starts at UEV voltage
- Added 2 more lutetium ways for the nuclear quest
- Added recipe gating info for mabs
![image](https://github.com/user-attachments/assets/d3cfd881-be06-4eb3-a828-f29c5550331e)

Closes https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/18071
Closes https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/18083
Closes https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/18088
Closes https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17348

chochem commented:
> Make zpm+ circuit assembler optional due to CAL CA mode

that is CAL problem and not a questbook problem. The stated intention of CA mode was to not skip any circuit assemblers

chochem commented on config/betterquesting/DefaultQuests/Quests/Tier11UIV-ytf2QCzUSk6ihcYeRbwblw==/TheDarknessofCho-AAAAAAAAAAAAAAAAAAAJbQ==.json:
```diff
@@ -30,7 +30,7 @@
       "autoClaim:1": 0,
       "isSilent:1": 0,
-      "desc:8": "In order to get to the riches you rightfully deserve, you\u0027ll first need to make a Raw Portal, then complete it with more materials. You can then access this dimension that should contain every ore imaginable...\n\n[note]Unfortunately, those tasty ores are locked beneath the mantle of the planet. You\u0027ll need to use a Void Miner to get them out.[/note]"
+      "desc:8": "\nIn order to get to the riches you rightfully deserve, you\u0027ll first need to make a Raw Portal, then complete it with more materials. You can then access this dimension that should contain every ore imaginable...\n\n[note]Unfortunately, those tasty ores are locked beneath the mantle of the planet. You\u0027ll need to use a Void Miner to get them out.[/note]\n\n[note]This sho ...
```
capitalize the Tartarite

chochem commented on config/betterquesting/DefaultQuests/Quests/BeeBreeding-AAAAAAAAAAAAAAAAAAAAEw==/Indium-AAAAAAAAAAAAAAAAAAABhg==.json:
```diff
@@ -46,7 +46,7 @@
       "autoClaim:1": 0,
       "isSilent:1": 0,
-      "desc:8": "You can not use the mutatron, and you have to go to Venus. But once you have it this bee is very strong!"
+      "desc:8": "This bee multiplies all sources of indium by 3x.\nIndium bee is available in LuV, or in IV with the help of magic."
```
`Indium` capitalized and a `The` missing at the start of the second sentence.

StaffiX commented:
Alright i removed the circuit assembler thing and fixed those two capitalization and missing word issues.
(also moved two quests and added a missing dependency)

chochem reviewed (approved):
thanks, that is great!

serenibyss reviewed (approved):
All lategame changes look accurate, thanks!
